packages/odyssey-react/src/components/Button.tsx

```diff
@@ -0,0 +1,58 @@
+/*!
```

Comment:
Note: This is missing icon support... because an Icon component doesn't exist yet.